src/Maestro/DependencyUpdateErrorProcessor/DependencyUpdateErrorProcessor.cs

```diff
@@ -203,7 +203,7 @@ public async Task ProcessDependencyUpdateErrorsAsync()
                                 break;
                             }
 
-                            _logger.LogInformation($"Matching issue {issueNumber} for {repoBranchUpdateHistoryError.Repository}:{repoBranchUpdateHistoryError.Branch} was closed. Creating a new issue.");
+                            _logger.LogInformation($"Matching issue {issueNumber.Value} for {repoBranchUpdateHistoryError.Repository}:{repoBranchUpdateHistoryError.Branch} was closed. Creating a new issue.");
```

Comment:
Really these should use the "structured logging" thing, and not interpolated strings, then the "issueNumber" would be available as a detail of the log.
```suggestion
                            _logger.LogInformation("Matching issue {IssueNumber} for {Repository}:{Branch} was closed. Creating a new issue.", issueNumber, repoBranchUpdateHistoryError.Repository, repoBranchUpdateHistoryError.Branch);
```
Nullables should ToString just fine, I think.

Reply:
In this form it doesn't stringify properly, that's why I added this. Though I guess I could add a good `.ToString()` to the class.

Reply:
Interpolated logging strings are present all over these classes, so I wanted to just limit the scope of changes. I didn't see much value in including _all_ interpolation-to-structured logging changes in this PR, but will if you think it's worth it.

Reply:
At least for ones you're touching, updating them is cheap and can't hurt.

Reply:
Went ahead and covered this whole class.